modules/osd-create-cluster-ccs.adoc

```diff
@@ -279,6 +279,13 @@ ifdef::osd-on-gcp[]
 If you are installing a cluster into a Shared VPC, the VPC name and subnets are shared from the host project.
 ====
 endif::osd-on-gcp[]
+ifdef::osd-on-aws[]
+.. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the machine pools created by default. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.
```

Comment:
```suggestion
.. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the machine pools that are created by default. You must have already created the security groups and associated them with the VPC that you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.
```
- From [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences#grammar): "Include the word _that_ in clauses unless it results in writing that sounds too formal or stilted. The use of the conjunction _that_, although technically optional in some sentences, is never wrong and makes the sentence easier to translate and clearer for readers whose primary language is not English."

Reply:
Found here also: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc